src/util.jl

```diff
@@ -92,3 +92,9 @@ TrivialView(a::AbstractArray{T,N}) where {T,N} = TrivialView{typeof(a),T,N}(a)
 # """
 @inline drop_sdims(a::StaticArray) = TrivialView(a)
 @inline drop_sdims(a) = a
+
+@static if VERSION >= v"0.7.0-DEV.3256"
+    const _pointer_from_objref = pointer_from_objref
+else
+    const _pointer_from_objref = Base.data_pointer_from_objref
+end
```

Comment:
With the previous comment in mind... I assume we can just use the public `pointer_from_objectref` in julia 0.6 as well as 0.7?

Reply:
Yeah I think you're right. I thought the additional type assertion maybe had a function, but judging by https://github.com/JuliaLang/julia/commit/1750a83ce233151d29101961db9de10e74a1f624 it was just redundant.